packages/stencil-component/src/utils/utils.ts

```diff
@@ -1,37 +1,39 @@
-import { Howl } from 'howler';
-import { BehaviorSubject, Subject } from 'rxjs';
+import {Howl} from 'howler';
+import {BehaviorSubject, Subject} from 'rxjs';
+import {Alignment, Page} from "../index.ds";
 
-
-export interface Page {
-  id: string,
-  paragraphs: Node[],
-  img?: string,
-  attributes?:NamedNodeMap[]
-}
-
-export interface Alignment {
-  [id: string]: [number, number];
+/**
+ * Inspect path
+ * @param path
+ * @return boolean
+ */
+export function looksLikeRelativePath(path: string): boolean {
+  return !(/^(https?:\/|assets|data:)\/\b/).test(path);
 }
 
 /**
- * Gets XML from path
+ * Gets Document from path
  * @param {string} path - the path to the xml file
  */
-function getXML(path: string): string {
```

Comment:
this function is removed entirely in dev.readalong-component, as it's only used in one place now